openpype/modules/deadline/plugins/publish/submit_publish_job.py

```diff
@@ -861,7 +870,8 @@ def process(self, instance):
             "jobBatchName": data.get("jobBatchName", ""),
             "useSequenceForReview": data.get("useSequenceForReview", True),
             # map inputVersions `ObjectId` -> `str` so json supports it
-            "inputVersions": list(map(str, data.get("inputVersions", [])))
+            "inputVersions": list(map(str, data.get("inputVersions", []))),
+            "colorspace": instance.data['colorspace']
```

Comment:
It seems pretty bold that `colorspace` key exists in `instance.data`. Or is there a plugin that's ensuring that key is there? :) I haven't seen that before in `instance.data` to begin with.

Reply:
Same say "bold", same say "stupid" :D.